benches/bench.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(plugin, test)]
+#![feature(plugin,const_fn,test)]
```

Comment:
I'm confused as to why this is necessary, given we are calling this fn ast::Constness::NotConst?